packages/core/src/multiline-input/MultilineInput.css

```diff
@@ -243,7 +243,7 @@
 /* Styles for button adornment */
 .saltMultilineInput-startAdornmentContainer > .saltButton,
 .saltMultilineInput-endAdornmentContainer > .saltButton {
-  --saltButton-padding: calc(var(--salt-spacing-50) - var(--salt-size-border));
+  --saltButton-padding: calc(var(--salt-spacing-50) - var(--salt-size-fixed-100));
```

Comment:
As this is padding, should it be `--salt-spacing-fixed-100`?

Reply:
Same inside border calculation